Bouncer/Models/FilterStore/FilterStoreFile.swift

```diff
@@ -83,6 +83,23 @@ extension FilterStoreFile {
         }.eraseToAnyPublisher()
     }
     
+    func update(filter: Filter) -> AnyPublisher<Void, Never> {
```

Comment:
If for any reason the publisher fails, we should return a proper error, so this publisher could potentially fail with a FilterStoreError

Reply:
Yeah, I thought "if I just picked this item from the list, why would it fail? but I guess error handling is a must.